plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManager.java

```diff
@@ -39,6 +39,9 @@
     public static final ConfigKey<Boolean> s_vmwareCleanOldWorderVMs = new ConfigKey<Boolean>("Advanced", Boolean.class, "vmware.clean.old.worker.vms", "false",
             "If a worker vm is older then twice the 'job.expire.minutes' + 'job.cancel.threshold.minutes' , remove it.", true, ConfigKey.Scope.Global);
 
+    static final ConfigKey<Integer> s_vmwareOVAPackageTimeout = new ConfigKey<Integer>(Integer.class, "vmware.package.ova.timeout", "Advanced", "3600",
```

Comment:
Config does not matches with the description in your comment.

Reply:
corrected the description